recipes/search_ncbi/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set version = "0.1.1" %}
+
+package:  
+  name: search_ncbi  
+  version: {{ version }}
+
+source:  
+  url: https://pypi.io/packages/source/s/search_ncbi/search_ncbi-0.1.1.tar.gz  
```

yamllint

[error] trailing spaces

(trailing-spaces)